### ISSUE TITLE
SBML export from RuleBender not simulating

### ISSUE THREAD

**Issue by paulflang** [AMICI-dev/AMICI#1332]
**What did you expect to happen?**
I could simulate the following file in COPASI, but it did not work with AMICI
[cell_cycle_v3.0.0_sbml.txt](https://github.com/AMICI-dev/AMICI/files/5513442/cell_cycle_v3.0.0_sbml.txt)

**What has happened instead?**
I got the following error:
```
model_tools/utils.py:27: in simulate_sbml
    sbmlImporter.sbml2amici(model_name=module_name, output_dir=temp_dir)
venv/lib/python3.8/site-packages/amici/sbml_import.py:322: in sbml2amici
    self._process_sbml(constant_parameters)
venv/lib/python3.8/site-packages/amici/logging.py:194: in wrapper_timer
    rval = func(*args, **kwargs)
venv/lib/python3.8/site-packages/amici/sbml_import.py:364: in _process_sbml
    self._gather_locals()
venv/lib/python3.8/site-packages/amici/logging.py:194: in wrapper_timer
    rval = func(*args, **kwargs)
venv/lib/python3.8/site-packages/amici/sbml_import.py:428: in _gather_locals
    self._gather_dependent_locals()
venv/lib/python3.8/site-packages/amici/sbml_import.py:480: in _gather_dependent_locals
    self.local_symbols[r.getId()] = self._sympy_from_sbml_math(
venv/lib/python3.8/site-packages/amici/sbml_import.py:1296: in _sympy_from_sbml_math
    _check_unsupported_functions(formula,
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

sym = times(_rateLaw1), expression_type = 'kineticLaw', full_sym = times(_rateLaw1)

    def _check_unsupported_functions(sym: sp.Expr,
                                     expression_type: str,
                                     full_sym: sp.Expr = None):
        """
        Recursively checks the symbolic expression for unsupported symbolic
        functions
    
        :param sym:
            symbolic expressions
    
        :param expression_type:
            type of expression, only used when throwing errors
        """
        if full_sym is None:
            full_sym = sym
    
        unsupported_functions = [
            sp.functions.factorial, sp.functions.ceiling, sp.functions.floor,
            sp.function.UndefinedFunction
        ]
    
        unsupp_fun_type = next(
            (
                fun_type
                for fun_type in unsupported_functions
                if isinstance(sym.func, fun_type)
            ),
            None
        )
        if unsupp_fun_type:
>           raise SBMLException(f'Encountered unsupported expression '
                                f'"{sym.func}" of type '
                                f'"{unsupp_fun_type}" as part of a '
                                f'{expression_type}: "{full_sym}"!')
E           amici.sbml_import.SBMLException: Encountered unsupported expression "times" of type "<class 'sympy.core.function.UndefinedFunction'>" as part of a kineticLaw: "times(_rateLaw1)"!

venv/lib/python3.8/site-packages/amici/sbml_import.py:1443: SBMLException
---------------------------------------------------------------------------------------------- Captured stdout call -----------------------------------------------------------------------------------------------
...
------------------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------------------
WARNING  py.warnings:warnings.py:109 /media/sf_DPhil_Project/Project04_Full cell cycle modelling/cell_cycle_model/venv/lib/python3.8/site-packages/sympy/__init__.py:672: SymPyDeprecationWarning: 

importing sympy.core.function with 'from sympy import *' has been
deprecated since SymPy 1.6. Use import sympy.core.function instead.
See https://github.com/sympy/sympy/issues/18245 for more info.

```

**To Reproduce**
*Steps to reproduce the behavior*
*Ideally include minimal code examples here*
```Python
import amici
import numpy as np
import os
import sys
from tempfile import TemporaryDirectory

def simulate_sbml(path, time=np.linspace(0, 70500, 101)):
    sbmlImporter = amici.SbmlImporter(path)
    with TemporaryDirectory() as temp_dir:
        model_name = os.path.splitext(os.path.basename(path))[0]
        module_name = model_name.replace('.', '_')
        sbmlImporter.sbml2amici(model_name=module_name, output_dir=temp_dir)
        sys.path.insert(0, temp_dir)
        model_module = amici.import_model_module(module_name=module_name,
                                                 module_path=temp_dir)
        model = model_module.getModel()
        model.setTimepoints(time)
        solver = model.getSolver()
        rdata = amici.runAmiciSimulation(model, solver)
        
        return rdata

rdata_x = simulate_sbml('cell_cycle_v3.0.0_sbml.xml'),
                                time = [0, 1])['x']
```

**AMICI version and system environment**
 - OS and version: [e.g. Ubuntu, iOS, Windows]: Ubuntu 20.04 VirtualBox on Windows10 host
 - AMICI interface: [e.g. Python, Matlab, C++]: Python 3.8.5
 - AMICI version: 0.11.8
 - Additional information: Compiler name and version used, Python/Matlab version, ...

**How to fix**
Do you know how to resolve the problem? No. I tried to import the file into and export from COPASI, but got a different error.
Can you submit a pull request? No

**Comment by FFroehlich** [comment]
putative fix in https://github.com/AMICI-dev/AMICI/pull/1333
Could you try whether that works for you? (Can be installed via `pip3 install -e git+https://github.com/AMICI-dev/AMICI@master#egg=amici\&subdirectory=python/sdist`

**Comment by dweindl** [comment]
> putative fix in #1333
> Could you try whether that works for you? (Can be installed via `pip3 install -e git+https://github.com/AMICI-dev/AMICI@master#egg=amici\&subdirectory=python/sdist`

`pip3 install -e git+https://github.com/AMICI-dev/AMICI@ef4338bedbefbe9e8311e70d912c8498eb22d83d#egg=amici\&subdirectory=python/sdist` - right?

**Comment by FFroehlich** [comment]
> > putative fix in #1333
> > Could you try whether that works for you? (Can be installed via `pip3 install -e git+https://github.com/AMICI-dev/AMICI@master#egg=amici\&subdirectory=python/sdist`
> 
> `pip3 install -e git+https://github.com/AMICI-dev/AMICI@ef4338bedbefbe9e8311e70d912c8498eb22d83d#egg=amici\&subdirectory=python/sdist` - not `master`, right?

sorry yes, corrected link, both should work now.

**Comment by paulflang** [comment]
Perfect. That seems to work. Thanks @FFroehlich  and @dweindl !
